csrankings.csv

```diff
@@ -13028,3 +13028,18 @@ Xiaodong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507/,AQewL04AAAAJ
 Xiao-Dong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507/,AQewL04AAAAJ
 Xun Yi,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/y/yi-professor-xun,oDRazncAAAAJ
 Zahir Tari ,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/t/tari-professor-zahir,Pz9Z7C0AAAAJ
+Jim Alves-Foss,University of Idaho,http://www2.cs.uidaho.edu/~jimaf,dOaERUAAAAAJ
+Daniel Conte de Leon,University of Idaho,http://www.cs.uidaho.edu/,aLH-yuoAAAAJ
+Michael Haney,University of Idaho,http://www.cs.uidaho.edu,yFPTY8wAAAAJ
+Robert B. Heckendorn,University of Idaho,http://marvin.cs.uidaho.edu/~heckendo/,Hd0x1I4AAAAJ
+Robert E. Hiromoto,University of Idaho,http://www.cs.uidaho.edu,NOSCHOLARPAGE
+Hasan M. Jamil,University of Idaho,http://www.cs.uidaho.edu,3x4_nY4AAAAJ
+Clinton L. Jeffery,http://www.cs.uidaho.edu,NOSCHOLARPAGE
```

Comment:
All of the entries below this point and including this one are ill-formed (they don't include the institution name). There are also two spurious newlines. Finally, the checklist should be updated to indicate that you have read and followed the instructions. Thanks.

Reply:
The raw URL http://www.cs.idaho.edu is also not suitable as a homepage entry for faculty.